metapool/calculate_coefficients.py

```diff
@@ -0,0 +1,147 @@
+import scipy
+import seaborn as sns
+import pandas as pd
+import numpy as np
+
+
+def plot_with_fit(xcol, ycol, groups, xmax=4, ymax=4,
+                  textpos=(.6, .2)):
+    from matplotlib import pyplot as plt
+    import matplotlib
+    matplotlib.use("TkAgg")
+    x = 0
+    y = 0
+    fig, axes = plt.subplots(xmax, ymax, sharex=True, sharey=True)
+    for name, group in groups:
+        ax = axes[y, x]
+        # ax.scatter(group["CPMlog"], group["Dilution2"])
+
+        # TODO FIXME HACK: Unclear what R^2 value is being printed by ggplot
```

Comment:
Is there something specific we need to do here? If this was a development comment, could you remove?

Reply:
The R code creates plots with ggplot2 and annotates them with R^2 values that cannot be replicated with python tools.  Someone verifying plots will see different values.  Unless someone comes back and says the new plots are fine, I'm uncomfortable removing the comment saying why the values are different.

Reply:
@dhakim87 Respectfully, if we're not sure on the behavior we shouldn't merge the code yet, IMHO. This is a production module used by the wet-lab.

Reply:
@charles-cowart Agreed though for different reasons.  This code isn't hooked up to anything and doesn't have the necessary input files yet to hook up to.  I was told to make a PR rather than a zip file to ease the review process.  Until the necessary files to hook into exist, I don't think it should be merged into production.  

I believe my role should end with porting the R code, not with negotiating APIs and file types with the wetlab.  

IMO we should create a branch that allows this feature to be picked up by @antgonza once there is something to hook into.  Would that be more amenable?  


On the ggplot2 differences: ggplot2's rendering is different from seaborns for regression plots (scatter plots with confidence interval of regression parameters) but the displayed regression lines aren't the ones that are used in the applied models.  Frankly if we're not going to be looking at the plots, I'd rather remove them from the python version entirely.

Reply:
@dhakim87 fair enough. I've created a dev branch off biocore:master. If you don't mind, can you redirect this PR to biocore:dev instead? Thanks!

Reply:
@charles-cowart Redirected to dev, thanks!